### DISCUSSION TITLE
RTL 502 Bad Gateway

### DISCUSSION THREAD

**Discussion by Casper0113** [Ride-The-Lightning/RTL#957] [Troubleshooting Questions]
I recently purchased a NODL DOJO and am unable to open RTL.  I continue to get a 502 Bad Gateway error.  I have tried stopping and starting the service, and refreshing to no avail.  I also tried accessing via the IP address instead of the default local address. 

LND and Bitcoin Core are running, and I have no issues with getting to the Lightning Terminal.  Any ideas?  I'm new to working with computers, so sorry for not providing additional information.  Any help is appreciated.

**Comment by ShahanaFarooqui** [comment]
@Casper0113 can you please share the RTL-Config.json file after removing password. The file must be in RTL’s root folder.

can you also share the screenshot of the screen for 502?

**Comment by Casper0113** [comment]
Dumb question, but can I access the json file from my laptop, or do I need to hook a monitor and keyboard up to the NODL?   I was able to hook up a monitor, but I only have a wireless keyboard and the NODL was not recognizing the keyboard, so I am unable to troubleshoot directly from the node.  

Sorry, I know it's like trying to explain physics to a monkey.  I assume the root folder is stored on the NODL hard drives since everything came pre-installed.  

NODL's software shows Ride The Lightning as "Running", but when I click the Open RTL button I get this error, no other data is on the screen. 

502 Bad Gateway
nginx/1.14.0 (Ubuntu)

Thanks for your help!

**Reply by saubyk** [reply]
Hi @Casper0113 have you tried reaching out to NODL support? they should be able to help you out relatively quickly.

Tagging @ketominer for visibility.

**Reply by Casper0113** [reply]
Yes, but I have not heard back yet.  Apologies for posting in the wrong section!

**Reply by saubyk** [reply]
no worry. I would recommend waiting for them to respond, before we get too down into the weeds to troubleshoot this. You can also reach out on their Telegram channel.
I will try to ping @ketominer as well.
